src/LocalStore.as

```diff
@@ -4,40 +4,69 @@ package
 
     public class LocalStore
     {
-        public static function getVariable(name:String, defaultValue:*):*
+        /** Local Shared Object, this is saved/flushed automatically when the application closes. */
+        private static var SO_OBJECT:SharedObject;
         {
-            var gameSave:SharedObject = SharedObject.getLocal(Constant.LOCAL_SO_NAME);
-            if (gameSave.data[name] != null)
-            {
-                return gameSave.data[name];
-            }
-            return defaultValue;
+            SO_OBJECT = SharedObject.getLocal(Constant.LOCAL_SO_NAME);
         }
```

Comment:
Is there a reason the `SO_OBJECT` assignment is put in braces?
What about `private static var SO_OBJECT:SharedObject = SharedObject.getLocal(Constant.LOCAL_SO_NAME);`?

Reply:
I didn't even check if you could assign a static var with a straight function call and not a static init for the shared object.
Though internally the way I assign it in the PR and using the suggested `private static var SO_OBJECT:SharedObject = SharedObject.getLocal(Constant.LOCAL_SO_NAME);`  would result in the same code.